CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ All notable changes to `src-cli` are documented in this file.
 
 ### Fixed
 
+- Execution URL when running `src batch remote` doesn't result in a 404. [sourcegraph/src-cli](https://github.com/sourcegraph/src-cli/pull/787)
```

Comment:
I think a user would have trouble parsing this, mostly because I had trouble parsing it. 😄 

How about:

```suggestion
- The preview link shown when running `src batch remote` to create a new batch change no longer 404s. [sourcegraph/src-cli](https://github.com/sourcegraph/src-cli/pull/787)
```

Reply:
Thanks. I was thinking about the best way to phrase that actually. hehe